test/__snapshots__/integration.test.js.snap

```diff
@@ -132,6 +132,7 @@ public class User {
 
 exports[`template integration tests using the generator should generate a comment for a consumer receiving multiple messages 1`] = `
 "
+
 import java.util.function.Consumer;
```

Comment:
Some snapshots are fine, some have an extra space now. I can't be bothered to endlessly fiddle with spacing for every use case. One extra space sometimes is within my tolerance. Let me know if you think we must make the spacing perfect.

Reply:
I see no need to be picky here. It's better to make it easy to work with.